Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -86,10 +93,12 @@ struct FPTIBatchData: Codable {
         }
 
         enum CodingKeys: String, CodingKey {
+            case buttonRank = "button_rank"
```

Comment:
Do these tags already exist in FPTI? If not we will need to work with the FPTI team to get them added before they will populate.

Reply:
It looks like the tag `experiment` already exists so we should be good there. Lets use `rank` which is already an approved tag. Once that is updated we should be able to verify in FPTI that we are seeing these tags as expected.

Reply:
ahhh gotcha, thanks for checking Jax! i've updated to use the already existing tags in cae7541467d37562712c34fe32e350c6778d939a